packages/common/container-definitions/src/runtime.ts

```diff
@@ -173,6 +177,9 @@ export interface IContainerContext {
 		summaryOp: ISummaryContent,
 		referenceSequenceNumber?: number,
 	) => number;
+	/**
+	 * @param content - Expected to be of type {@link @fluidframework/core-interfaces/internal#ISignalEnvelope}.
```

Comment:
Can we add a semantic description of `submitSignalsFn` while we're here? 😁

Reply:
(Feel free to close with no action - I just have a compulsive need to ask for more documentation 😜)